aws/karpenter-core
# feat: Add Node Requirement Drift

Fixes #N/A 

**Description**
- Karpenter will be able to detect machines that have been drifted from the Node Requirements of its provisioner
- Implements drift for Node requirements.

**How was this change tested?**
- Manually tested
- Added e2e testing (https://github.com/aws/karpenter/pull/3933)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
